analytic_engine/src/compaction/picker.rs

```diff
@@ -420,8 +477,17 @@ impl SizeTieredPicker {
 
 /// Time window compaction strategy
 /// See https://github.com/jeffjirsa/twcs/blob/master/src/main/java/com/jeffjirsa/cassandra/db/compaction/TimeWindowCompactionStrategy.java
```

Comment:
Modify comment?

Reply:
Already comment at L182 to explain current behavior, we can update this when new strategy is stable.